daemon/transmission-daemon.1

```diff
@@ -101,7 +101,7 @@ Enable distributed hash table (DHT).
 Disable distribued hash table (DHT).
 .It Fl p Fl -port Ar port
 Port to open and listen for RPC requests on. Default: 9091
-.It Fl P, -peerport Ar port
+.It Fl P -peerport Ar port
```

Comment:
I presume there should be a `Fl` instead of a comma, to be in line with the rest of the option specifications.

Reply:
Good catch!